tests/EscaladeTestCase.php

```diff
@@ -59,15 +58,61 @@ protected function logOut()
         $_SESSION['glpi_currenttime'] = $ctime;
     }
 
-    public function climbWithTimelineButton(\Ticket $ticket, \Group $group, array $options): void
+    /**
+     * Get the methods for simulating different ways of escalating a ticket.
+     *
+     * @param array $methods Contains the names of the different methods that must be used to simulate an escalation.
+     * eg. ['climbWithTimelineButton', 'climbWithHistoryButton'] to simulate the escalation with the timeline and history buttons.
+     *
+     * @return array
+     */
+    public static function climbTicketMethods(array $methods = []): array
```

Comment:
For the functions and variable names you add, replace “climb” with “escalate” for greater accuracy.
```suggestion
    public static function escalateTicketMethods(array $methods = []): array
```
If you wish, you can also correct the existing ones, but in another PR.